src/js/main.js

```diff
@@ -363,6 +363,30 @@ if (document.body.dataset.page === "integrations") {
   }
 }
 
+// Add underline class to menu links on Mobile
+
+function setActiveMenuItem() {
+  const currentUrl = window.location.href;
+  const menuItems = document.querySelectorAll("nav a");
+
+  menuItems.forEach((item) => {
+    const itemHref = item.getAttribute("href");
+    const itemUrl = new URL(itemHref, currentUrl);
+
+    const isIntegrationsPage = itemUrl.pathname === "/integrations.html";
```

Comment:
Note that on the actual website Netlify rewrites the URLs and there is no `.html` suffix, you can try in the deployment preview :)